src/cluster_legacy.c

```diff
@@ -7717,6 +7789,12 @@ const char **clusterCommandExtendedHelp(void) {
         "LINKS",
         "    Return information about all network links between this node and its peers.",
         "    Output format is an array where each array element is a map containing attributes of a link",
+        "MIGRATESLOTS SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...] NODE <node id>",
```

Comment:
would this be the right syntax to indicate that the user can repeat the slot range/node pair?
```suggestion
        "MIGRATESLOTS [SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...] NODE <node id>]+",
```

Reply:
Actually, the way it is written, it is:

SLOTSRANGE <start> <end> <start> <end>

Similar to ADDSLOTSRANGE. So you shouldn't repeat the SLOTSRANGE token.

Reply:
I don't think we use this regex-like syntax elsewhere. Is this standard?

Reply:
probably not to this extent but [FAILOVER](https://github.com/valkey-io/valkey/pull/1949/files#diff-2515500619600c5a1e7a8d9aaa8761a6071314cd226c3b7ee4d0e054691f0a8eR7771) is one close example.

Reply:
Looking again, my understanding is that these are documented how they appear in valkey-cli, which is:

```
CLUSTER MIGRATESLOTS SLOTSRANGE start-slot end-slot [start-slot end-slot ...] NODE node-id [SLOTSRANGE start-slot end-slot [start-slot end-slot ...] NODE node-id ...]
```

So I will document it the same